scss/_stepped-process.scss

```diff
@@ -0,0 +1,139 @@
+//
+// Orange stepped process
+//
+.stepped-process {
+  ol {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    overflow: hidden;
+    counter-reset: step;
+  }
+}
+
+.stepped-process-item {
+  display: flex;
+  flex: 1;
+  padding: map-get($spacers, 2);
+  margin-right: $stepped-process-margin-r;
+  font-weight: $font-weight-bold;
+  text-align: center;
+  list-style: none;
```

Comment:
I think those could move up to `ol` declaration above. Need to check, but it'd be simpler to understand.